.github/workflows/ci.yml

```diff
@@ -68,6 +68,26 @@ jobs:
           DATADOG_API_KEY: ${{ secrets.datadog_api_key }}
           DATADOG_APP_KEY: ${{ secrets.datadog_app_key }}
 
+  standalone-binary-test:
+    name: Test standalone binary
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '10.24.1'
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '2.7'
+      - run: npm config set python python2.7
+      - run: yarn install --immutable
+      - run: yarn build
+      - name: Bundle standalone binary
+        run: yarn dist-standalone -t 'linux-x64-14.15.3'
```

Comment:
Nexe does not have a `10.24.1` version, though I think we should bundle the package with the same version of node than the one used to install dependencies/build the package.
Either we should reconsider the node version we use for development or settle on using `14.15.3` for the bundled package only. (and continue testing the package with 10, 12, 14 and 16 versions)

Reply:
I'll use `14.15.3` for the bundled package for the moment. We can revisit the other options later